docker-compose.yml

```diff
@@ -755,17 +757,15 @@ services:
         base: ${R_ORG}/${R_IMAGE}:${R_TAG}
     shm_size: *shm-size
     environment:
-      TEST_R_WITH_ARROW: "true"
       NOT_CRAN: "true"
       LIBARROW_DOWNLOAD: "false"
+      ARROW_HOME: "/arrow"
+      ARROW_USE_PKG_CONFIG: "FALSE"
```

Comment:
This env var is the converse of `LIBARROW_BUILD: false`, as in: ignore any arrow C++ libraries installed on the system and do the bundled build in the R package installation.

Reply:
Ok. Any reason it's uppercase "FALSE"? Is it case-sensitive?

Reply:
Most of them aren't but I think this one is, seemed like it would be tedious to make it case insensitive in the configure script (while the others are handled in R and wrapped in `tolower()`) but it's probably worth it so as not to get tripped up on it.